db/compaction_picker.cc

```diff
@@ -156,9 +91,9 @@ void CompactionPicker::ReleaseCompactionFiles(Compaction* c, Status status) {
   }
 }
 
-void CompactionPicker::GetRange(const CompactionInputFiles& inputs,
```

Comment:
If you don't think it is helpful, I can revert it. The code base is new to me, so your input about which one is easier to understand is more important than mine.